[2024-10-15T15:12:27.553+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-15T15:12:27.589+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v0.task1 manual__2024-10-15T15:12:26.570506+00:00 [queued]>
[2024-10-15T15:12:27.609+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v0.task1 manual__2024-10-15T15:12:26.570506+00:00 [queued]>
[2024-10-15T15:12:27.610+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-15T15:12:27.629+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task1> on 2024-10-15 15:12:26.570506+00:00
[2024-10-15T15:12:27.633+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_backfill_v0', 'task1', 'manual__2024-10-15T15:12:26.570506+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_backfill.py', '--cfg-path', '/tmp/tmpv1c4elq2']
[2024-10-15T15:12:27.635+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask task1
[2024-10-15T15:12:27.635+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=487) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-15T15:12:27.636+0000] {standard_task_runner.py:72} INFO - Started process 488 to run task
[2024-10-15T15:12:27.701+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_backfill_v0.task1 manual__2024-10-15T15:12:26.570506+00:00 [running]> on host 49b50434baff
[2024-10-15T15:12:27.776+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='james' AIRFLOW_CTX_DAG_ID='dag_with_catchup_backfill_v0' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T15:12:26.570506+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T15:12:26.570506+00:00'
[2024-10-15T15:12:27.777+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-15T15:12:27.795+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-15T15:12:27.795+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is awesome']
[2024-10-15T15:12:27.800+0000] {subprocess.py:86} INFO - Output:
[2024-10-15T15:12:27.801+0000] {subprocess.py:93} INFO - This is awesome
[2024-10-15T15:12:27.801+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-15T15:12:27.836+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-15T15:12:27.836+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v0, task_id=task1, run_id=manual__2024-10-15T15:12:26.570506+00:00, execution_date=20241015T151226, start_date=20241015T151227, end_date=20241015T151227
[2024-10-15T15:12:27.889+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-15T15:12:27.919+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T15:12:27.921+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
